### DISCUSSION TITLE
Featured in Batocera content downloader

### DISCUSSION THREAD

**Discussion by Hew-ux** [DarrenCarol/MrOverlay#11] [General]
Hey there, Atari from Batocera here. These are some pretty neat bezels. A user asked if we could add these to our built-in content downloader (where you can download things such as freeware games and other bezels). I don't see a reason why not, so I'm putting the packs onto our store's list.

Let me know if you explicitly don't want this or have any reason to forbid such distribution.

**Comment by DarrenCarol** [comment]
Hi Atari,
I'm more than happy for these to be added, thank you for letting me know.

Many thanks,
Darren

**Reply by Hew-ux** [reply]
Awesome. Let me know if you make any updates, as the package on the store is static. Alternatively, you can create the package yourself if you wish and just upload newer versions directly to the Discord: https://wiki.batocera.org/create_pacman_package
